terminal/print.go

```diff
@@ -5,21 +5,21 @@ import (
 )
 
 var (
-	ansiStdout = NewAnsiStdout()
+	AnsiStdout = NewAnsiStdout()
```

Comment:
Yea i noticed, nice idea - since the `Ansi` bit is a bit of an artifact of the old package, how do you think about just naming this `Stdout`? I know I probably make some dumb naming request on every PR (sorry!) but the API is already a bit verbose, i think it would be good to try to minimize it wherever possible